scipy/stats/kde.py

```diff
@@ -69,6 +72,8 @@ class gaussian_kde(object):
         Number of dimensions.
     n : int
         Number of datapoints.
+    neff : int
+        Effective number of datapoints.
```

Comment:
Could you add a blank line and then a line with `.. versionadded:: 1.2.0` below this?

Reply:
yes, indeed. forgot to point that out